ui/src/main/js/application/auth/AuthenticationConfiguration.js

```diff
@@ -77,6 +84,23 @@ const AuthenticationConfiguration = ({
     const hasLdapConfig = Object.keys(AUTHENTICATION_LDAP_FIELD_KEYS).some((key) => FieldModelUtilities.hasValue(formData, AUTHENTICATION_LDAP_FIELD_KEYS[key]));
     const hasSamlConfig = Object.keys(AUTHENTICATION_SAML_FIELD_KEYS).some((key) => FieldModelUtilities.hasValue(formData, AUTHENTICATION_SAML_FIELD_KEYS[key]));
 
+    const importBlackDuckSSOConfigLabel = "Retrieve Black Duck SAML Configuration";
```

Comment:
I think our linter prefers single quotes here. This should fix itself with a fix current file command